File: problem_forty_two.py
#Modular Exponentiation (Power in Modular Arithmetic)
inp =int(input("Enter the number:"))
power = int(input("Enter the power:"))
mod = int(input("Enter the mod:"))
def MyFunction(inp,power,mod):
    value = (inp**power)%mod
    return value

print("Result = ",MyFunction(inp,power,mod))    

File: problem_forty_two.js
//Modular Exponentiation (Power in Modular Arithmetic)
var number = 2
var power = 10
var mod = 5
function myFunction(number,power,mod){
    var value = (number**power)%mod
    return value
}
console.log("Result = ",myFunction(number,power,mod))